src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LINE = "---------------------------------------------------------";
+    private static Task taskList[] = new Task[100];
```

Comment:
Could call this `tasks` too, but I like the naming `taskList` more (don't tell the TAs)